slash/fun.py

```diff
@@ -33,6 +33,15 @@ async def headsortails(self, interaction: Interaction):
         embed = nextcord.Embed(color=0x0DD91A, title=f"It is {hot}!")
         await interaction.response.send_message(embed=embed)
 
+    @slash(guild_ids=SLASH_GUILDS)
+    async def f(self, interaction: Interaction, text: str = SlashOption(required=True)):
```

Comment:
Please add a description to the `text` parameter (`SlashOption(description=..., required=True)`)

Reply:
i will fix this in the next commit in combination with your second requested change